Makefile

```diff
@@ -37,7 +15,6 @@ DOCKER_ENVS := \
 	-e DOCKER_CLIENTONLY \
 	-e DOCKER_DEBUG \
 	-e DOCKER_EXPERIMENTAL \
-	-e DOCKERFILE \
```

Comment:
It should still be overwritable in the same way via something like `make DOCKERFILE=Dockerfile.gccgo gccgo` just like before -- this variable being passed in to the container (which is what this line was doing) was controlling where the "frozen images" were being pulled from if they weren't available in the image already, which likely would've been broken with a different `DOCKERFILE=...` anyhow.